src/utils/isEmailValid.ts

```diff
@@ -1,22 +1,20 @@
+import { z } from 'zod';
```

Comment:
`zod` is imported only as a dependency from the `next` package in our `package-lock.json`. But I would suggest adding all packages used in our code directly in our `package.json` with `npm add zod` and not rely on it being loaded by some dependency!

Reply:
Btw. you did add the package `zod` in https://github.com/Plant-for-the-Planet-org/planet-donations/pull/440 ;-)

Reply:
Did that myself now.